src/main/java/com/minecolonies/coremod/entity/pathfinding/PathNavigate.java

```diff
@@ -349,7 +349,7 @@ protected void pathFollow()
                   && !pExNext.isOnLadder())
             {
                 final Vec3d vec3 = getEntityPosition();
-                if ((vec3.yCoord - (double) pEx.yCoord) < 0.001)
+                if ((vec3.y - (double) pEx.y) < 0.001)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 0.001 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)